### ISSUE TITLE
cfg_query subcommand adds extra empty and non-empty lines to the output

### ISSUE THREAD

**Issue by porst17** [hilbert/hilbert-cli#67]
Take a look at the query
```
$ python3 tools/hilbert.py cfg_query --configfile tests/data/Hilbert.yml -o Stations/testhost1/address
test1.host.dns.name
...

```
The output should just be
```
test1.host.dns.name
```
but it contains a line with `...` for (most?) single line results and an extra empty line in any case.

**Comment by malex984** [comment]
should be fixed with PR #69

**Comment by malex984** [comment]
still to be tested / improved

**Comment by porst17** [comment]
Fixed via #69
